includes/session.php

```diff
@@ -152,10 +152,11 @@
 define('WT_START_TIME', microtime(true));
 
 // We want to know about all PHP errors during development, and fewer in production.
+// In production, do not overwrite server-defined error reporting.
 if (WT_DEBUG) {
-	error_reporting(E_ALL | E_STRICT | E_NOTICE | E_DEPRECATED);
+	error_reporting(-1);
 } else {
-	error_reporting(E_ALL);
+	error_reporting(error_reporting() & E_ALL & ~E_DEPRECATED);
```

Comment:
`E_DEPRECATED` makes sense.

Do we also want `error_reporting()`?  If there are errors, then we want to display/log them.